### PR TITLE
#135 : separate the header naming convention from parameterNamingConvention

### PR DESCRIPTION
**Pull request by mohamedelhabib**

### CODE REVIEW

File: lib/src/main/java/org/openapitools/openapistylevalidator/NamingValidator.java [hunk 1]
```diff
@@ -29,12 +36,22 @@ private boolean isSeparatorCaseValid(String variableName, String separator) {
                 return false;
             }
             totalLength += token.length();
-            if (!token.toLowerCase().equals(token)) {
-                return false;
-            }
-
-            if (!token.matches(REGEX_LOWER_CASE_ALPHA_NUMERIC_ONLY)) {
-                return false;
+            if(isUpperCase) {
```
Comment (JFCote) [inline lib/src/main/java/org/openapitools/openapistylevalidator/NamingValidator.java:39]: please do a "reformat" here or juste add a space between `if` and `(`.

**Review by JFCote — CHANGES_REQUESTED** [review]
Only a small missing space and everything else looks good to me.

**Review by JFCote — APPROVED** [review]
LGTM. Will merge shortly

**Comment by mohamedelhabib** [comment]
@JFCote Thanks for the quick merge. 
Did you know when this will be released ?

**Comment by JFCote** [comment]
@jmini is the only one currently able to do a release. I have no problem doing one after this PR has passed CI: https://github.com/OpenAPITools/openapi-style-validator/pull/104

So @jmini let me know over slack when you think you'll be able to do a release.

**Comment by jmini** [comment]
@mohamedelhabib:

I have released version `1.5` of `openapi-style-validator`
https://github.com/OpenAPITools/openapi-style-validator/releases/tag/1.5

**Pull request merged**